qt/scientific_interfaces/Inelastic/Analysis/FunctionBrowser/ConvFunctionModel.cpp

```diff
@@ -11,12 +11,34 @@
 #include "MantidAPI/MultiDomainFunction.h"
 #include "MantidQtWidgets/Common/FunctionBrowser/FunctionBrowserUtils.h"
 
+#include <map>
+
+namespace {
+using namespace MantidQt::CustomInterfaces::IDA;
+
+auto const lorentzian = [](Mantid::MantidVec const &x, Mantid::MantidVec const &y) {
+  (void)x;
+  return std::unordered_map<std::string, double>{{"Amplitude", y[1]}};
+};
+
+auto const sqeFunction = [](Mantid::MantidVec const &x, Mantid::MantidVec const &y) {
+  (void)x;
+  return std::unordered_map<std::string, double>{{"Height", y[1]}};
```

Comment:
Fit functions that are derived from `IPeakFunction` have `setHeight` methods etc. that could avoid some of this hard-coding of the parameter names, but I note the SQE function are not peak functions - maybe they could be at some point in the future? note they would also need getters and setters for fwhm and centre.

Reply:
SQE functions have a Lorentzian peak in their equations

Reply:
The peak width is a function of Q (spectrum) and fitting parameters such as Diffusion Coeff , tau, beta